{
  try {
    return c.call();
  }
 catch (  Exception e) {
    throw new IgniteCheckedException(e);
  }
}
